go.mod

```diff
@@ -195,6 +195,13 @@ require (
 )
 
 require (
+	github.com/alecthomas/colour v0.1.0 // indirect
+	github.com/alecthomas/repr v0.0.0-20210801044451-80ca428c5142 // indirect
+	github.com/mattn/go-isatty v0.0.20 // indirect
+)
+
+require (
+	github.com/alecthomas/assert v1.0.0
```

Comment:
Were you not able to find any similar unit testing pattern here without including a new assertion lib?